src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="main-container">
+      <div class="container">
+        <div class="day-cell day-cell--start-day_mon day-cell--1"></div>
+        <div class="day-cell day-cell--2"></div>
+        <div class="day-cell day-cell--3"></div>
+        <div class="day-cell day-cell--4"></div>
+        <div class="day-cell day-cell--5"></div>
+        <div class="day-cell day-cell--6"></div>
+        <div class="day-cell day-cell--7"></div>
+        <div class="day-cell day-cell--8"></div>
+        <div class="day-cell day-cell--9"></div>
+        <div class="day-cell day-cell--10"></div>
+        <div class="day-cell day-cell--11"></div>
+        <div class="day-cell day-cell--12"></div>
+        <div class="day-cell day-cell--13"></div>
+        <div class="day-cell day-cell--14"></div>
+        <div class="day-cell day-cell--15"></div>
+        <div class="day-cell day-cell--16"></div>
+        <div class="day-cell day-cell--17"></div>
+        <div class="day-cell day-cell--18"></div>
+        <div class="day-cell day-cell--19"></div>
+        <div class="day-cell day-cell--20"></div>
+        <div class="day-cell day-cell--21"></div>
+        <div class="day-cell day-cell--22"></div>
+        <div class="day-cell day-cell--23"></div>
+        <div class="day-cell day-cell--24"></div>
+        <div class="day-cell day-cell--25"></div>
+        <div class="day-cell day-cell--26"></div>
+        <div class="day-cell day-cell--27"></div>
+        <div class="day-cell day-cell--28"></div>
+        <div class="day-cell day-cell--29"></div>
+        <div class="day-cell day-cell--30"></div>
+        <div class="day-cell day-cell--31"></div>
```

Comment:
You can make days an elements of container block. Also you can you use nth-child() in CSS instead of making a new modifier to each day, but it`s up to you.